tree/tree/src/TTree.cxx

```diff
@@ -427,32 +427,48 @@ static char DataTypeToChar(EDataType datatype)
 {
    // Return the leaflist 'char' for a given datatype.
 
-   switch(datatype) {
-   case kChar_t:     return 'B';
-   case kUChar_t:    return 'b';
-   case kBool_t:     return 'O';
-   case kShort_t:    return 'S';
-   case kUShort_t:   return 's';
-   case kCounter:
-   case kInt_t:      return 'I';
-   case kUInt_t:     return 'i';
-   case kDouble_t:
-   case kDouble32_t: return 'D';
-   case kFloat_t:
-   case kFloat16_t:  return 'F';
-   case kLong_t:     return 0; // unsupported
-   case kULong_t:    return 0; // unsupported?
-   case kchar:       return 0; // unsupported
-   case kLong64_t:   return 'L';
-   case kULong64_t:  return 'l';
-
-   case kCharStar:   return 'C';
-   case kBits:       return 0; //unsupported
-
-   case kOther_t:
-   case kNoType_t:
-   default:
-      return 0;
+   switch (datatype) {
```

Comment:
Hi, Philippe. 

astyle. Still haven't tried hard enough to make the clang_format file work with VS Code.